### PR TITLE
Warning and codegen cleanups

### PR DESCRIPTION
**Pull request by mullr**
This PR isolates the cleanup commits from #145

### CODE REVIEW

File: generate_modules.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/env bash
```
Comment (nerdrew) [inline generate_modules.sh]: `/usr/bin/env`

**Comment by nerdrew** [comment]
Ooops. I'm still learning the repo. It looks like the tests still failed. It looks like we should remove all the `+stable`'s from the bash scripts. Mind adding something like this commit to your PR? https://github.com/tafia/quick-protobuf/commit/e9268f30d65bed411b7371e9a2ab8207bbd6863d

**Comment by nerdrew** [comment]
FWIW green build with my commit on this branch: https://travis-ci.org/tafia/quick-protobuf/builds/582760010

**Comment by mullr** [comment]
Cherry picked that onto this branch

**Comment by nerdrew** [comment]
Thanks! Want to rebase your `no_std` PR (that or wait a little bit for me to open a PR to gitignore most of the generated files and rebase on both)?

**Comment by mullr** [comment]
If your gitignore pr is imminent I'll just rebase on that.

**Comment by nerdrew** [comment]
PR here: https://github.com/tafia/quick-protobuf/pull/147 want to review?

**Pull request merged**